FindOnPage.md

```diff
@@ -0,0 +1,653 @@
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT StartFind(
+      [in] ICoreWebView2StagingFindConfiguration* configuration
+      , [in] ICoreWebView2StagingFindStartFindCompletedHandler* handler
+  );
+
+  /// Navigates to the next match in the document.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT FindNext(
+  );
+
+  /// Navigates to the previous match in the document.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT FindPrevious(
+  );
+
+  /// Stops the current 'Find' operation and hides the Find bar.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  HRESULT StopFind(
+  );
+
+
+}
+
+
+
+/// Interface defining the find configuration.
+/// This interface provides the necessary methods and properties to configure a find operation.
+// MSOWNERS: core (maxwellmyers@microsoft.com)
+[uuid(52a04b23-acc8-5659-aa2f-26dbe9faafde), object, pointer_default(unique)]
+interface ICoreWebView2StagingFindConfiguration : IUnknown {
+  /// Gets the `FindDirection` property.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT FindDirection([out, retval] COREWEBVIEW2_FIND_DIRECTION* value);
+
+
+  /// 
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT FindDirection([in] COREWEBVIEW2_FIND_DIRECTION value);
+
+
+  /// Gets the `FindTerm` property.
+  ///
+  /// The caller must free the returned string with `CoTaskMemFree`.  See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings).
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT FindTerm([out, retval] LPWSTR* value);
+
+
+  /// 
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT FindTerm([in] LPCWSTR value);
+
+
+  /// Gets the `IsCaseSensitive` property.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT IsCaseSensitive([out, retval] BOOL* value);
+
+
+  /// 
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT IsCaseSensitive([in] BOOL value);
+
+
+  /// Gets the `ShouldMatchWord` property.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT ShouldMatchWord([out, retval] BOOL* value);
+
+
+  /// 
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT ShouldMatchWord([in] BOOL value);
+
+
+
+}
+
+///  
+/// Interface providing methods to access the find operation functionalities in the CoreWebView2.
+/// 
+// MSOWNERS: core (maxwellmyers@microsoft.com)
+[uuid(c9a130ca-a807-549c-9d76-8e09ccee3973), object, pointer_default(unique)]
+interface ICoreWebView2Staging17 : IUnknown {
+  /// Retrieves the find operation interface for the current web view.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT Find([out, retval] ICoreWebView2StagingFind** value);
+
+
+
+}
+```
+
+
+### Setting Up Find Configuration with MIDL3
+
+### CoreWebView2 Find Configuration and Direction
+
+```csharp
+namespace Microsoft.Web.WebView2.Core
```

Comment:
You can remove this C# section lines 499-515. We just want the COM IDL which you have above and the MIDL3 which you have below.